src/components/LoadingIndicator/LoadingIndicator.tsx

```diff
@@ -7,29 +7,19 @@ import classNames from 'classnames';
 
 export type LoadingIndicatorProps = {
   className?: string;
-  text?: string | ReactNode;
+  message?: ReactNode;
   spinnerPosition?: 'bottom' | 'left' | 'right';
 };
 
-export const TEST_ID = {
-  CONTAINER: 'loading-indicator-container',
-  SPINNER: 'loading-indicator-spinner',
-  TEXT: 'loading-indicator-text'
-};
-
-export const LoadingIndicator = ({ className, text, spinnerPosition = 'bottom' }: LoadingIndicatorProps) => {
+export const LoadingIndicator = ({ className, message, spinnerPosition = 'bottom' }: LoadingIndicatorProps) => {
   return (
-    <div
-      className={classNames('zui-loading-indicator', `zui-loading-indicator-${spinnerPosition}`, className)}
-      data-testid={TEST_ID.CONTAINER}
-    >
-      <Spinner data-testid={TEST_ID.SPINNER} />
-      {text && (
+    <div className={classNames('zui-loading-indicator', `zui-loading-indicator-${spinnerPosition}`, className)}>
+      <Spinner />
+      {message && (
         <div
-          className={classNames('zui-loading-indicator-text', `zui-loading-indicator-text-${spinnerPosition}`)}
-          data-testid={TEST_ID.TEXT}
+          className={classNames('zui-loading-indicator-message', `zui-loading-indicator-message-${spinnerPosition}`)}
         >
-          {text}
+          {message}
```

Comment:
We only need to apply the positioning classes to the `zui-loading-indicator` container.

Reply:
Ah Yeah you're right, I should have noticed this wouldn't have been changing anything. Updated!